docs/source/hooks/examples.md

```diff
@@ -30,10 +30,6 @@ class MemoryProfilingHooks:
     def __init__(self):
```

Comment:
Can't comment this high up in the code, but...

1. Can we add the remaining imports? I think we're just missing `from kedro.framework.hooks import hook_impl`; seems silly that the docs leave out that one necessary import.
2. Can you format the imports as follows, just to be consistent with Black/isorted code style (why not):

    ```python
    import logging

    from kedro.framework.hooks import hook_impl
    from memory_profiler import memory_usage
    
    
    class MemoryProfilingHooks:
    ```
    
Some other inconsistencies, not sure what want to do about it:

* Some examples below put a comment with the filename at the top of the script.
* This example seems to be hidden in a "Click to expand" block, while others in the same doc aren't.

Reply:
@deepyaman no problem, see in [12fc10f](https://github.com/kedro-org/kedro/pull/2257/commits/12fc10f9985d3cd392d597ded0f45746460623d3)

1. added
2. re-ordered
3. added for all examples (1&4 examples were missing the comment)
4. removed to be consistent with the rest examples in the same doc

Cheers